feat(changelog): elastic-metal-added-debian-12-"bookworm"-is 2023-06-14 (#1805)
scaleway/docs-content: changelog/june2023/2023-06-14-elastic-metal-added-debian-12-bookworm-is.mdx

---

> title: Debian 12 Bookworm is now available for your bare metal servers

Reviewer: ```suggestion
title: Debian 12 Bookworm is now available for your Elastic Metal servers
```

status: added
author:
    fullname: 'Join the #elastic-metal channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2023-06-14
category: compute
product: elastic-metal
---

> You can now find the just released Debian 12 Bookworm when installing your Elastic Metal server.

Reviewer: ```suggestion
You can now deploy your Elastic Metal servers with Debian 12 Bookworm.
```
